### ISSUE TITLE
acme.sh does not recognise existing proper TXT record

### ISSUE THREAD

**Issue by naox** [Neilpang/acme.sh#1375]
```
./acme.sh  --issue -d *.a1.x25.pl --dns dns_cf --debug
```

```
[śro mar 14 14:06:31 CET 2018] d='*.a1.x25.pl'
[śro mar 14 14:06:31 CET 2018] _d_alias
[śro mar 14 14:06:31 CET 2018] txtdomain='_acme-challenge.a1.x25.pl'
[śro mar 14 14:06:31 CET 2018] txt='7XrPLgZ7-FgAKS99k3nz5DYUO4D-K5_A3s4DLSxCGtg'
[śro mar 14 14:06:31 CET 2018] d_api
[śro mar 14 14:06:31 CET 2018] Add the following TXT record:
[śro mar 14 14:06:31 CET 2018] Domain: '_acme-challenge.a1.x25.pl'
[śro mar 14 14:06:31 CET 2018] TXT value: '7XrPLgZ7-FgAKS99k3nz5DYUO4D-K5_A3s4DLSxCGtg'
[śro mar 14 14:06:31 CET 2018] Please be aware that you prepend _acme-challenge. before your domain
[śro mar 14 14:06:31 CET 2018] so the resulting subdomain will be: _acme-challenge.a1.x25.pl
[śro mar 14 14:06:31 CET 2018] Dns record not added yet, so, save to /home/user/.acme.sh/*.a1.x25.pl/*.a1.x25.pl.conf and exit.
[śro mar 14 14:06:31 CET 2018] Please add the TXT records to the domains, and retry again.
[śro mar 14 14:06:31 CET 2018] pid
[śro mar 14 14:06:31 CET 2018] No need to restore nginx, skip.
[śro mar 14 14:06:31 CET 2018] _clearupdns
[śro mar 14 14:06:31 CET 2018] skip dns.
[śro mar 14 14:06:31 CET 2018] _on_issue_err
[śro mar 14 14:06:31 CET 2018] Please add '--debug' or '--log' to check more details.
[śro mar 14 14:06:31 CET 2018] See: https://github.com/Neilpang/acme.sh/wiki/How-to-debug-acme.sh
[śro mar 14 14:06:31 CET 2018] socat doesn't exists.
[śro mar 14 14:06:31 CET 2018] Diagnosis versions:
```

```
# dig _acme-challenge.a1.x25.pl TXT +trace +short @8.8.8.8
TXT "7XrPLgZ7-FgAKS99k3nz5DYUO4D-K5_A3s4DLSxCGtg" from server ns1.x25.pl in 1 ms.
```

**Comment by Neilpang** [comment]
你确定安装了所有文件了吗  还是只拷贝了 一个 acme.sh 文件.
dns_cf.sh 没有找到.

**Comment by naox** [comment]
Now i've copied every file. Thats not the issue however as I want to use dns in manual mode.

I've aplied fix #1375 and problem persists:

```
[śro mar 14 14:29:03 CET 2018] d='*.a2.x25.pl'
[śro mar 14 14:29:03 CET 2018] _d_alias
[śro mar 14 14:29:03 CET 2018] txtdomain='_acme-challenge.a2.x25.pl'
[śro mar 14 14:29:03 CET 2018] txt='g1cLIzi2uguxjua0890DJHd6CT_RaWu7s4MKXo1GpA4'
[śro mar 14 14:29:03 CET 2018] d_api
[śro mar 14 14:29:03 CET 2018] Add the following TXT record:
[śro mar 14 14:29:03 CET 2018] Domain: '_acme-challenge.a2.x25.pl'
[śro mar 14 14:29:03 CET 2018] TXT value: 'g1cLIzi2uguxjua0890DJHd6CT_RaWu7s4MKXo1GpA4'
[śro mar 14 14:29:03 CET 2018] Please be aware that you prepend _acme-challenge. before your domain
[śro mar 14 14:29:03 CET 2018] so the resulting subdomain will be: _acme-challenge.a2.x25.pl
[śro mar 14 14:29:03 CET 2018] Dns record not added yet, so, save to /home/naox/.acme.sh/*.a2.x25.pl/*.a2.x25.pl.conf and exit.
[śro mar 14 14:29:03 CET 2018] Please add the TXT records to the domains, and retry again.
[śro mar 14 14:29:03 CET 2018] pid
[śro mar 14 14:29:03 CET 2018] No need to restore nginx, skip.
[śro mar 14 14:29:03 CET 2018] _clearupdns
[śro mar 14 14:29:03 CET 2018] skip dns.
[śro mar 14 14:29:03 CET 2018] _on_issue_err
[śro mar 14 14:29:03 CET 2018] Please add '--debug' or '--log' to check more details.
[śro mar 14 14:29:03 CET 2018] See: https://github.com/Neilpang/acme.sh/wiki/How-to-debug-acme.sh
[śro mar 14 14:29:03 CET 2018] socat doesn't exists.
[śro mar 14 14:29:03 CET 2018] Diagnosis versions:
openssl:openssl
```

```
# dig '_acme-challenge.a2.x25.pl' TXT +trace +short @8.8.8.8                                             NS a.root-
TXT "g1cLIzi2uguxjua0890DJHd6CT_RaWu7s4MKXo1GpA4" from server ns3.x25.pl in 1 ms.
```

**Comment by Neilpang** [comment]
手动模式需要分两步,  添加完 txt 记录之后 需要 调用renew

```
./acme.sh  --issue -d *.a2.x25.pl --dns --debug

./acme.sh  --renew  -d *.a2.x25.pl  --debug
```
